CONTRIBUTING.md

```diff
@@ -10,12 +10,119 @@ We actively welcome your pull requests.
+6. Install all required packages with `npm install`
+7. Then run `npm run all` to set up Rapid.
+8. Finally, run `npm run quickstart` to start the server on port 8080.
+
+Congrats! 🎉 You should now be able to use rapid by navigating to [http://localhost:8080/](http://localhost:8080/) on Chrome.
+
+### Setting up the VSCode debugger
+VSCode provides a debugging mode using Chrome. To use it, do this:
+
+1. Go to **Run>Add Configuration** and select `{} Chrome: Launch`
+2. This will create `launch.json`.
+3. Make sure to add the `/dist/` path to `webRoot` so that it looks like this:
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}/dist/"
+        }
+    ]
+}
+```
+
+Great! You should now be able to run the Rapid server with `npm run quickstart` and then the debugger by either pressing **F5** on your keyboard, or clicking on **Run>Start Debugging**.
+
+This will launch Chrome on whatever address you provided to `url` on `launch.json`, allowing you to use Rapid as well as entering debugging mode on VSCode.
+
+> **Note:** Again, you are welcome to try your own debugging method, such as [Chrome DevTools](https://developer.chrome.com/docs/devtools/javascript/breakpoints/).
+
+### Debugging example
+
+When hacking Rapid, it will be useful to watch the code execute line-by-line. To do this, you can set up inline breakpoints. You can learn all about it in [the official documentation](https://code.visualstudio.com/docs/editor/debugging).
+
+A good place to start is the `/index.html` file. Set a breakpoint on the `context.initAsync();` call from it's last few lines:
+
+```javascript
+...
+          window.context = context;  // for debugging
+          context.initAsync();
+        }
+      }
+    </script>
+  </body>
+</html>
+```
+Then run `npm run quickstart` and press **F5** to launch Chrome along with the VSCode debugger.
+
+This will freeze execution right at the moment of Rapid initialization. Click on the **Step in** button on the debugger controls (the arrow pointing down into a point) to enter the `context.initAsync()` function. Then click on **Step over** repeatedly to watch the code execute line-by-line.
+
+This will be a good primer on what's going on under the hood of Rapid. You can set up breakpoints anywhere in the code to figure out the call stack, live changes in variable values, or any other detail you may need.
+
+Please refer to [the official documentation](https://code.visualstudio.com/docs/editor/debugging) for more on all the information you can get from debugging.
```

Comment:
Consider changing the wording here, something like:

> Please refer to [the official documentation](https://code.visualstudio.com/docs/editor/debugging) for more information on debugging in VS Code.

Reply:
Changed this to 

Please see [the official documentation](https://code.visualstudio.com/docs/editor/debugging) to learn all the details debugging can show you.